### PR TITLE
Added error message when running neovim with true colors

### PR DESCRIPTION
**Pull request by regnat**
See #16

### CODE REVIEW

File: autoload/limelight.vim [hunk 1]
```diff
@@ -32,6 +32,11 @@ set cpo&vim
 let s:default_coeff = str2float('0.5')
 let s:invalid_coefficient = 'Invalid coefficient. Expected: 0.0 ~ 1.0'
 
+function! s:nvim_unsupported()
+  let var = 'g:limelight_conceal_guifb'
```
Comment (junegunn) [inline autoload/limelight.vim]: Should be `fg` not `fb`, right?

File: autoload/limelight.vim [hunk 2]
```diff
@@ -32,6 +32,11 @@ set cpo&vim
...
+  return 'Neovim with true colors is unsupported yet. Please set '.var
```
Comment (junegunn) [inline autoload/limelight.vim]: I think `is not yet supported` sounds more idiomatic.

**Comment by junegunn** [comment]
Thanks for looking into this. I left a couple of comments on the commit. Could you update it accordingly?

**Comment by regnat** [comment]
Thanks for the review, I've updated the pr

**Comment by junegunn** [comment]
Sorry for being too picky, but can you squash the commits into one? Also, I use present tense when writing commit messages, so `Add` instead of `Added`. Thanks!

**Comment by regnat** [comment]
Yes, of course, I know it's important to keep a clean repo

**Comment by junegunn** [comment]
Thanks!

**Pull request merged**